addons/l10n_ec/views/l10n_ec_sri_payment.xml

```diff
@@ -1,12 +1,33 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <odoo>
-    <record id="account_l10n_ec_sri_payment" model="ir.ui.view">
-        <field name="name">l10n.ec.sri.payment.form</field>
+
+    <record id="view_payment_method_form" model="ir.ui.view">
+        <field name="name">view.payment.method.form</field>
         <field name="model">l10n_ec.sri.payment</field>
+        <field name="type">form</field>
         <field name="arch" type="xml">
-            <tree>
-                <field name="name"/>
+            <form string="Payment Method" create="0" edit="0">
+                <sheet>
+                    <group>
+                          <field name="code"/>
+                          <field name="name"/>
```

Comment:
Move all of this to l10n_ec_edi

Reply:
@vbe-odoo We could close the comment, it seems we all agree on how it is right now, it won't break anything